packages/test-sync/src/commonTest/kotlin/io/realm/kotlin/test/mongodb/common/serializer/BsonEncoderTests.kt

```diff
@@ -143,13 +141,6 @@ class BsonEncoderTests {
                             bsonValue = BsonBinary(BsonBinarySubType.UUID_STANDARD, uuid.bytes),
                         )
                     }
-                ObjectId::class -> ObjectId.create().let { objectId ->
```

Comment:
Shouldn't we then have a test for `BsonObjectId`? I guess this is a test of our addition to the serializers, so maybe not required, but we are testing it for `BsonDecimal128`. https://github.com/realm/realm-kotlin/pull/1748/files#diff-57eed610740b3e30ea24c8fe99231780bb1e17c15dbcc0dd3adb9d5654b3a803R64 has a comment that `BsonObjectId` is tested elsewhere, but I can't really see how that should be the case from following the code. 

Since serialization is just reusing kbson's serializer I don't think it is a blocker, but more of a nice to have.